RELEASE_NOTES.md

```diff
@@ -1,3 +1,72 @@
+# ChefDK 4.1 Release Notes
+
+## Updated Components
+
+### Chef Infra Client 15.1
+
+Chef Infra Client has been updated to 15.1 with new and improved resources, improvements to target mode, bootstrap bug fixes, new Ohai detection on VirtualBox hosts, and more. See the [Chef Infra Client 15.1 Release Notes](https://github.com/chef/chef/blob/master/RELEASE_NOTES.md#chef-infra-client-151) for a complete list of new and improved functionality.
+
+### Chef InSpec 4.6.9
+
+Chef InSpec has been updated from 4.3.2 to 4.6.9 with the following changes:
+
+  - InSpec `Attributes` have now been renamed to `Inputs` to avoid confusion with Chef Infra attributes.
+  - A new InSpec plugin type of `Input` has been added for defining new input types. See the [InSpec Plugins documentation](https://github.com/inspec/inspec/blob/master/docs/dev/plugins.md#implementing-input-plugins) for more information on writing these plugins.
+  - InSpec no longer prints errors to the stdout when passing `--format json`.
+  - When fetching profiles from GitHub, the URL can now include periods.
+  - The performance of InSpec startup has been improved.
+
+### Cookstyle 5.0.0
+
+Cookstyle has been updated to 5.0.0 with a large number of bugfixes and major improvements that lay the groundwork for future autocorrecting of cookobook style and deprecation warnings.
+
+The RuboCop engine that powers Cookstyle has been updaed from 0.62 to 0.72, which includes several hundred bugfixes to the codebase. You may experience existing cookbooks, which now fail using Cookstyle 5.0 due to some of these bugfixes. Additionally some cops have had their names changed and the the Rubocop Performance cops have been removed. If you disabled individual cops in your .rubocop.yml file this may require you update that confg.
```

Comment:
```suggestion
The RuboCop engine that powers Cookstyle has been updated from 0.62 to 0.72, which includes several hundred bugfixes to the codebase. When using Cookstyle 5.0, you may experience existing cookbooks now failing, due to some of these bugfixes. Additionally, some cops have had their names changed and the Rubocop Performance cops have been removed. If you disabled individual cops in your .rubocop.yml file, this may require you update your confg.
```